app/assets/stylesheets/application.scss

```diff
@@ -12,3 +14,11 @@
     margin-top: 2em;
   }
 }
+.my-primary-btn, .my-primary-btn:active, .my-primary-btn:visited {
+    background-color: $primary-color !important;
+    color: white !important;
+}
+
+.my-primary-btn:hover {
+   background-color: $hover-primary-color !important;
+}
```

Comment:
Use nesting syntax of scss to integrate pseudo classes. E.g.
.button-primary {
     ....
     &:hover { ... }
     &:active { ... }
}

Rename my-primary-btn.